src/Coordination/KeeperStateManager.cpp

```diff
@@ -321,6 +323,11 @@ KeeperStateManager::KeeperStateManager(
     , keeper_context(keeper_context_)
     , logger(getLogger("KeeperStateManager"))
 {
+    if (keeper_context_->getCoordinationSettings()[CoordinationSetting::rotate_log_storage_interval] <= 0)
+    {
+        LOG_FATAL(logger, "Keeper parameter rotate_log_storage_interval must be greater than 0.");
+        std::terminate();
+    }
```

Comment:
Instead of terminating, I could catch this case and reset the parameter to the default.